docs/building.md

```diff
@@ -0,0 +1,158 @@
+# Building the code
+
+This document describes steps needed for compiling the code of the Smart Card
+Connector App and the examples of programmatic communication with this App.
```

Comment:
nit: I think 'programmatic communication' usually means something different than 'how two programs communicate with each other'. I would replace 'and the examples ...' with 'and examples of how programs can communicate with the app'.